### DISCUSSION TITLE
Could not find file 'C:\<deletedfile>'

### DISCUSSION THREAD

**Discussion by Hyguys** [dotnet/winforms#7572] [Q&A]
### Environment

Microsoft Visual Studio Community 2022 (64 bits) - Current Version 17.3.0

### .NET version

Not sure - the latest (I'm a beginner)

### Did this work in a previous version of Visual Studio and/or previous .NET release?

Not sure 

### Issue description

I was trying to make my solution to a template because just copying and pasting the solution folder was not working to open in a new computer - showing the error pipe name timed out DesignToolsServer. 
So I tried to make my solution a template then open this template in a new computer - which worked "better", only that when opening on the new computer, it tried to find a file that was in my other computer.
So I deleted the file on my old computer hoping that the solution could open without it (it was an image and I had already made a copy of it with a different name and put in on the solution folder). However, now I can't even open the solution on my own computer, because the file was deleted, but the solution keeps trying to find it.
![image](https://user-images.githubusercontent.com/40918340/183985716-5f446376-983a-4e43-82ab-b6bc0632615f.png)
![image](https://user-images.githubusercontent.com/40918340/183985780-2bf91d30-3631-42ac-a23e-523b4825a18c.png) (As you can see the image is not on the Resouce window).
How can I remove the reference to this image that doesn't exist anymore using visual studio community 2022?

### Steps to reproduce

Add a image to your project, attach it to a image control and delete it manually. Your designer should break

### Diagnostics

```text
em StreamJsonRpc.JsonRpc.d__143`1.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
em Microsoft.DotNet.DesignTools.Client.Host.ServerHost.d__16.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
em Microsoft.DotNet.DesignTools.Client.DesignToolsClientLoader.d__29.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
em Microsoft.DotNet.DesignTools.Client.DesignToolsClientLoader.<>c__DisplayClass25_1.<<-ctor>b__1>d.MoveNext()
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
em Microsoft.DotNet.DesignTools.Client.DesignToolsClientLoader.d__27.MoveNext()
```

**Comment by RussKie** [comment]
@Hyguys if I understood the issue correctly, you have added an image as a resource, but then you deleted the image file from the disk. The designer requires the physical file to be present, otherwise it can't render your form/control.
To fix the issue you need to manually modify the resx file and remove the reference to the deleted image file.
